src/Form/JanrainCaptureDrupalSettingsForm.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace Drupal\janrain_capture\Form;
+
+use Drupal\Core\Form\ConfigFormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Entity\Entity\EntityViewMode;
+
+/**
+ * Janrain Capture settings form for Drupal config.
+ */
+class JanrainCaptureDrupalSettingsForm extends ConfigFormBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function getEditableConfigNames() {
+    return ['janrain_capture.drupal_settings'];
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId() {
+    return 'janrain_capture.drupal_settings.capture';
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateInterface $form_state) {
+    $form = parent::buildForm($form, $form_state);
+    $config = $this->config('janrain_capture.drupal_settings');
+
+    $form['capture'] = [
+      '#type' => 'fieldset',
+      '#tree' => TRUE,
+      '#title' => $this->t('Janrain Capture Drupal settings'),
+    ];
+    $view_modes = \Drupal::entityQuery('entity_view_mode')
```

Comment:
Ideally, use Dependency Injection instead of a direct call to a container with `\Drupal::entityQuery`